packaging/linux/common/notes.appdata.xml.in

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 Ruby Mamistvalove -->
+<component type="desktop-application">
+    <id>@APP_ID@</id>
+    <launchable type="desktop-id">@APP_ID@.desktop</launchable>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>MPL-2.0</project_license>
+    <name>@PROJECT_NAME@</name>
+    <developer_name>Ruby Mamistvalove</developer_name>
+    <update_contact>ruby.mamistvalove_AT_gmail.com</update_contact>
```

Comment:
@nuttyartist Is this your actual, current email? I actually don't remember where I got it from... 😓

Reply:
Yes! It is!